src/WorkItems/FilingClientFactory.cs

```diff
@@ -5,18 +5,15 @@
 using System.Globalization;
 using System.Text.RegularExpressions;
 
-namespace Microsoft.WorkItemFiling
+namespace Microsoft.WorkItems
 {
     public class FilingClientFactory
     {
-        public static FilingClient CreateFilingTarget(string filingHostUriString)
+        public static FilingClient CreateFilingTarget(Uri filingUri)
```

Comment:
>filingUri [](start = 58, length = 9)

I agree with removing `String`, but I think `filingHostUri` is a better name, because it's assigned to the `HostUri` property of the `FilingClient`, and the URI you ultimately POST to will have this as a prefix, but this isn't the whole thing.

Reply:
Done. Also renamed this method to Create, so usage will be FilingClientFactory,Create.

---
In reply to: [381020374](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381020374) [](ancestors = 381020374)